apps/rush-lib/src/scripts/install-run.ts

```diff
@@ -452,9 +452,16 @@ export function installAndRun(
   const originalEnvPath: string = process.env.PATH || '';
   let result: childProcess.SpawnSyncReturns<Buffer>;
   try {
+    // Node.js on Windows can not spawn a file when the path has a space on it
+    // unless the path gets wrapped in a cmd friendly way and shell mode is used
+    const shouldUseShell: boolean = binPath.includes(' ') && os.platform() === 'win32';
+    const scapedBinPath: string = shouldUseShell ? `"${binPath}"` : binPath;
+
     process.env.PATH = [binFolderPath, originalEnvPath].join(path.delimiter);
-    result = childProcess.spawnSync(binPath, packageBinArgs, {
+    result = childProcess.spawnSync(scapedBinPath, packageBinArgs, {
       stdio: 'inherit',
+      windowsVerbatimArguments: false,
```

Comment:
Isn't this the default value already?

Reply:
It is but only when `shell` is false. When `shell` is true, `windowsVerbatimArguments` changes to true too. In this way it's always false.